eagle-jpm/eagle-hadoop-queue/src/main/java/org/apache/eagle/hadoop/queue/model/scheduler/Queue.java

```diff
@@ -24,7 +24,6 @@
 @JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Queue {
-    private String type;
```

Comment:
can you comment this field rather than removing it, also adding the explanation I provided in the EAGLE-561 ? that would be helpful.